include/clasp/core/lisp.h

```diff
@@ -379,6 +379,7 @@ class Lisp {
   bool _MpiEnabled;
   int _MpiRank;
   int _MpiSize;
+  int _TrapFpeBits;
```

Comment:
Is this field only used in the handle_fpe restoration code?

Reply:
Basically, yes.

Reply:
Alright, then please mention this in a comment and that's fine